avocado/utils/network/interfaces.py

```diff
@@ -75,6 +75,11 @@ def _get_interface_details(self, version=4):
             log.error(msg)
             raise NWException(msg)
 
+    def _get_bondinterface_details(self, version=4):
+        cmd = "cat /sys/class/net/{}/bonding/mode /sys/class/net/{}/bonding/slaves".format(self.name, self.name)
+        output = run_command(cmd, self.host)
+        return output
```

Comment:
Some interfaces don't have bonding directory. We need to handle here,  and return None instead of an exception.